packages/create-jest/src/runCreate.ts

```diff
@@ -7,10 +7,13 @@
 
 import * as path from 'path';
 import chalk = require('chalk');
+import exit = require('exit');
 import * as fs from 'graceful-fs';
 import prompts = require('prompts');
+import yargs = require('yargs');
```

Comment:
Just thinking out loud: it feels too much to ship `yargs` with this package. I think `process.argv[2]` would be to do the job. The value will be always `string | undefined`.

Currently this package is 15.5 kB (unpacked). `yargs` takes 292 kB (unpacked, without dependencies). That’s what npm gave to me.

Reply:
Sounds fair, I think keeping this package light is more important than having universal parsing for arguments and `--help` interface